website.js

```diff
@@ -319,6 +316,17 @@ io.on('connection', function (socket) {
             socket.emit('nextSong');
         }})
     })
+    socket.on('createPoll', function (data) {
+        connection.query('insert into poll set ?', data, function(err, result) {})
+    })
+    socket.on('getPoll', function (data) {
+        connection.query('select * from poll where id = ?', data, function(err, result) {
+            socket.emit('pollData', result[0].answers)
+        })
+    })
+    socket.on('addResult', function (data) {
+        connection.query('update poll set answers = ? where id = "' + data.id + '"', JSON.stringify(data.answers), function(err,result){})
```

Comment:
Great work! 🎉 if you have any questions, let me know! Happy to help! 👍

Also, feel free to let me know if you prefer less/more or different kind of feedback. Open for anything 👍